coffea/nanoevents/transforms.py

```diff
@@ -1,10 +1,19 @@
 import copy
 import numpy
 import numba
-import awkward1
+import awkward as ak
 from coffea.nanoevents.util import concat
 
 
+def data(stack):
```

Comment:
What kind of array does this get called on? pyarrow?
Most of the rest are specifically for awkward layout types (we should probably label them to indicate what type of input is valid)

Reply:
This shows up in from_buffers even for root files, `/data` is more or less equivalent to `/content`.

Reply:
From https://github.com/scikit-hep/awkward-1.0/pull/592/files#diff-e9f2e3c299511a19f2535f6fdaaca21cb69f3893a3d7fe57aa775ef7110e5317L3410-L3418 the NumpyArray form is now keyed with `data` where it used to not be keyed with any attribute. FYI @jpivarski was this intended? This is fine, though I think the NanoEvents DSL could just as well tack `!skip` after every `!content` rather than make this no-op call (would save some tiny amount of time)
What we can also do is change `key_formatter` to ignore any attribute arguments on the assumption that the form key already fully resolves the required attribute. Then we wouldn't need `!skip` or `!data` DSL commands